docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -105,97 +107,5 @@ The `tokenizer` parameter controls the tokenizers that will be used to
-
-[source,synonyms]
---------------------------------------------------
-include::{es-test-dir}/cluster/config/analysis/synonym.txt[]
---------------------------------------------------
-
-You can also define synonyms for the filter directly in the
-configuration file (note use of `synonyms` instead of `synonyms_path`):
-
-[source,console]
---------------------------------------------------
-PUT /test_index
-{
-  "settings": {
-    "index": {
-      "analysis": {
-        "filter": {
-          "synonym": {
-            "type": "synonym_graph",
-            "synonyms": [
-              "lol, laughing out loud",
-              "universe, cosmos"
-            ]
-          }
-        }
-      }
-    }
-  }
-}
---------------------------------------------------
-
-However, it is recommended to define large synonyms set in a file using
-`synonyms_path`, because specifying them inline increases cluster size unnecessarily.
-
-[discrete]
-==== WordNet synonyms
-
-Synonyms based on https://wordnet.princeton.edu/[WordNet] format can be
-declared using `format`:
-
-[source,console]
---------------------------------------------------
-PUT /test_index
-{
-  "settings": {
-    "index": {
-      "analysis": {
-        "filter": {
-          "synonym": {
-            "type": "synonym_graph",
-            "format": "wordnet",
-            "synonyms": [
-              "s(100000001,1,'abstain',v,1,0).",
-              "s(100000001,2,'refrain',v,1,0).",
-              "s(100000001,3,'desist',v,1,0)."
-            ]
-          }
-        }
-      }
-    }
-  }
-}
---------------------------------------------------
-
-Using `synonyms_path` to define WordNet synonyms in a file is supported
-as well.
-
-[discrete]
-==== Parsing synonym files
-
-Elasticsearch will use the token filters preceding the synonym filter
-in a tokenizer chain to parse the entries in a synonym file. So, for example, if a
-synonym filter is placed after a stemmer, then the stemmer will also be applied
-to the synonym entries. Because entries in the synonym map cannot have stacked
-positions, some token filters may cause issues here. Token filters that produce
-multiple versions of a token may choose which version of the token to emit when
-parsing synonyms, e.g. `asciifolding` will only produce the folded version of the
-token. Others, e.g. `multiplexer`, `word_delimiter_graph` or `ngram` will throw an
-error.
-
-If you need to build analyzers that include both multi-token filters and synonym
-filters, consider using the <<analysis-multiplexer-tokenfilter,multiplexer>> filter,
-with the multi-token filters in one branch and the synonym filter in the other.
-
-WARNING: The synonym rules should not contain words that are removed by
-a filter that appears after in the chain (a `stop` filter for instance).
-Removing a term from a synonym rule breaks the matching at query time.
+See <<synonyms-define-synonyms,define your synonyms>> for the available formats to define synonyms.
```

Comment:
Let's keep these files as they were in the original format!

Reply:
Why not updating these files with the new way of defining synonyms? I think it's important they are updated with how Synonyms API can be used, and it's an opportunity to remove some duplicated content here (synonyms format) and consolidate it in a single place. 

Is there a reason for not changing these files?